docs/proc/how_we_work_in_the_team.md

```diff
@@ -0,0 +1,30 @@
+# Bref
+Here you can find how we are working in the dev team and what dev team members
+should to help other members be on the same page.
+
+# Code contribution guide
+Please find and read document `docs/proc/code_contribution_guide.md`.
+
+# Releases
+Each repository SHOULD have a release tag, and each tag should have a strong formatted text, which is: MAJOR-MINOR-PATCH. For forked repos that we are changing SHOULD be: FORKED_REPOS_MAJOR-FORKED_REPOS_MINOR-FORKED_REPOS_PATCH-MAJOR-MINOR-PATCH.
+
+Example: let's say polkadotsdk has version 1.0.0, qf-solochain has version 0.5.0, then the full version is 1.0.0-0.5.0.
```

Comment:
What should we do with existing release tags?

Reply:
I guess we could ignore them. I think just small CI/CD changes if we start using this logic.

Reply:
Ok